guides.md

```diff
@@ -0,0 +1,83 @@
+# Guides
+
+Guides explain in practical terms how we do stuff. Any Hypha member can
+change these.<sup>[1](#references)</sup>
+
+We are inspired by the idea of "protocols" from the [Lab
+Book][clear-lab-book] of the [Civic Laboratory for Environmental Action
+Research (CLEAR)][clear]
+
+> Protocols are the "attitudes" or "the manner in which one approaches each and
+every element in our space." They are different than pure rules or
+instructions; they are ways that we establish order and maintain practices
+across our group [...] -- They define the way we ought to proceed or behave in
+different situations. As such, they are normative, or premised on values,
+morals, and an idea of how things ought to be done. They are a manifestation of
+our values.
+
+## Scheduling a meeting
+
+Note: Consider whether this meeting might be a global interest to other
+members. Skip below steps at your discretion for low-stakes topics.
+
+- Try to announce your intention to schedule a meeting on a topic, and ask
+  who's interested to participate. Allow a day for people to respond.
+- Consider who might be most interested, and what timing might work for
+  them.
+  - Check the [monthly `availability` calendar for travel/vacation plans][availability].
+  - Check everyone's [weekly availability `schedules`][schedules].
+- If 2-3 candidate timeslots feel obvious from looking at internal
+  availabilities:
+  - Reserve them all in [Hypha calendar][calendar] immediately, for example: `HOLD:
+    Infrastructure Meeting (Option 1)`
+  - Ask in chat whether anyone who wants to attend has any blocks on any
+    of the options.
+- If short-notice or speed of booking is important:
+  - Claim an open HOLD [:information_source:](/meetings.md#holds) slot.
+    - If time permits or topic isn't an all-hands matter, consider
+      leaving the hold available for someone else.
+    - If claiming a hold, edit the calendar immediately, and announce
+      meeting in chat.
+- If your must find a new timeslot:
+  - Create a new poll. (We currently use [When2Meet](https://www.when2meet.com/).)
```

Comment:
```suggestion
  - Create a new poll (We currently use [When2Meet](https://www.when2meet.com/)).
```

Reply:
oh jeez, my intuitions on this stuff are different. I believe punctuation is fine to go inside when the whole sentence is parenthetical, right?
https://english.stackexchange.com/a/11129/55389

I'm less inclined to do it your suggested way, unless it's of the structure "(and ...)" or "(when ... )". But as it stands, it's a full sentence in the parens :)

Thoughts?

Reply:
If it is a full sentence than it shouldn't be in a parenthetical, as I'm used to them explaining/qualifying and doing so while signalling it is set apart from of the flow of writing. This definition roughly corresponds to my understanding: https://www.vocabulary.com/dictionary/parenthetical